test/jdk/java/text/Format/NumberFormat/Bug8132125.java

```diff
@@ -26,20 +26,27 @@
  * @bug 8132125 8202537
  * @summary Checks Swiss' number elements
  * @modules jdk.localedata
+ * @run junit Bug8132125
  */
 
-import java.text.*;
-import java.util.*;
+import java.text.NumberFormat;
+import java.util.Locale;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class Bug8132125 {
-    public static void main(String[] args) {
+
+    // Ensure the CLDRConverter does not omit the Swiss number elements
+    @Test
+    public void swissNumElementsTest() {
         Locale deCH = Locale.of("de", "CH");
         NumberFormat nf = NumberFormat.getInstance(deCH);
 
-        String expected = "54\u2019839\u2019483.142"; // i.e. "\u2019" as decimal separator, "\u2019" as grouping separator
+        // i.e. "\u2019" as decimal separator, "\u2019" as grouping separator
```

Comment:
This is incorrect (sorry, I am to be blamed as I wrote this 😄). Should be "`\u002E` as decimal separator, " also remove `i.e.` if you move the comment on top of the variable declaration

Reply:
Fixed.